packages/astro/src/types/public/config.ts

```diff
@@ -1390,6 +1390,76 @@ export interface AstroUserConfig {
 	 */
 	legacy?: object;
 
+	/**
```

Comment:
Just checking on this position, because it seems like this would make it show up underneath "Legacy Flags". Did you mean to put it here, or after i18n?

(Can also be put in any position in the main list, as you think it seems appropriate.)